src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>header</title>
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <a href="#"><img src="./assets/images/logo.jpg" alt="logo"></a>
+    </div>
+    <nav>
+      <ul class="navList">
+        <li><a href="index.php">Special flavours</a></li>
+        <li><a href="about.php">By regions</a></li>
+        <li><a href="contact.php">By ingredients</a></li>
```

Comment:
The `nav links` take you to an empty page when clicked. Adding a # symbol before the link will prevent that from happening. Example; `href="#index.php"`.

Reply:
Done 👌